### DISCUSSION TITLE
PR #2027 causes issue as an import using esbuild.

### DISCUSSION THREAD

**Discussion by jbosse** [alpinejs/alpine#2342] [7. Bugs (archived)] [answered]
Upgrading to 3.5 has broken my app which uses alpinejs as an NPM dependency. I am using Phoenix/Elixir with esbuild. The issue seems to be caused by https://github.com/alpinejs/alpine/pull/2027/files specifically https://github.com/alpinejs/alpine/blob/253fd8fc14ad8b0c22864774286921d741975235/packages/alpinejs/src/evaluator.js#L94 which is raising an error `Alpine Expression Error: func(...).catch is not a function` when the page loads trying to process my `x-data` directive.

I am not sure how to create a simple, easy to run app that can demonstrate the issue and I cannot reproduce it using the CDN script in a CodePen, only when I use 
```
import Alpine from "alpinejs";

window.Alpine = Alpine;
Alpine.start();
```
inside my app.js file.

I assume it is something wonky in the transpiling from ES6.

**Comment by danddanddand** [comment, marked as answer]
See this thread... https://github.com/alpinejs/alpine/discussions/2318

**Reply by jbosse** [reply]
Yes thank you I didn't even know to look at the discussions tab until the new issue made a discussion instead 😬 TIL.

**Comment by SimoTod** [comment]
A few users reported it on discord. the standard target for phoenix is es2016 which does not support async function so func().catch is not valid. If you set the target to es2017, it will work.
